spec/models/manageiq/providers/vmware/infra_manager/metric_capture_spec.rb

```diff
@@ -5,8 +5,12 @@
     MiqRegion.seed
 
     guid, server, @zone = EvmSpecHelper.create_guid_miq_server_zone
+
+    Timecop.freeze(Time.parse("2011-08-12T23:33:00Z").utc)
```

Comment:
why this ancient date?  is it because it has to match something in a VCR?  if not, maybe we can make it date relative (e.g. yesterday or last week)

Reply:
> is it because it has to match something in a VCR

Yes it matches the counter timestamps, e.g. `sampleInfoCSV: 20,2011-08-12T20:33:20Z,20,2011-08-12T20:33:40Z,20` from https://github.com/ManageIQ/manageiq-providers-vmware/blob/eb70bd086e8a74178561773568be0dcba167644b/spec/tools/vim_data/miq_vim_perf_history/queryPerfMulti.yml#L1308

I wanted the datapoints that get returned to be realistic in case we start dropping counters returned outside the requested window in the future.

Reply:
yea - perfect example of where vcrs break down